packages/@jsii/go-runtime/jsii-runtime-go/helpers.go

```diff
@@ -2,23 +2,45 @@ package jsii
 
 import "time"
 
+type Type interface {
+	bool | string | float64 | time.Time
+}
+
+func V[T Type](v T) *T {
+	return &v
+}
+
+func Slice[T Type](v ...T) *[]*T {
```

Comment:
Missing doc-string. It also feels to me the name should somehow indicate it has similar semantics to `V` above.

Reply:
So, in this case, the name `Ptr` for a single value, and `PtrSlice` for a slice?

Or `Value` and `Values`?